sync/sync-lib/src/main/cpp/ddg-sync-jni.c

```diff
@@ -70,6 +70,37 @@ Java_com_duckduckgo_sync_crypto_SyncNativeLib_prepareForLogin(
     return result;
 }
 
+JNIEXPORT jint JNICALL
+Java_com_duckduckgo_sync_crypto_SyncNativeLib_encrypt(
+    JNIEnv *env,
+    jclass clazz,
+    jbyteArray encryptedBytes,
+    jbyteArray rawBytes,
+    jbyteArray secretKey
+) {
+    jsize rawBytesLength = (*env)->GetArrayLength(env, rawBytes);
+
+    // Get pointers to the arrays
+    jbyte* encryptedBytesElements = (*env)->GetByteArrayElements(env, encryptedBytes, NULL);
+    jbyte* rawBytesElements = (*env)->GetByteArrayElements(env, rawBytes, NULL);
+    jbyte* secretKeyElements = (*env)->GetByteArrayElements(env, secretKey, NULL);
+
+    // Call the C function
+    jint result = ddgSyncEncrypt(
+      (unsigned char *)encryptedBytesElements,
+      (unsigned char *)rawBytesElements,
+      (unsigned long long)rawBytesLength,
+      (unsigned char *)secretKeyElements
+    );
+
+    // Release the input arrays
+    (*env)->ReleaseByteArrayElements(env, encryptedBytes, encryptedBytesElements, JNI_ABORT);
```

Comment:
if this is the value you want returned, you should not use JNI_ABORT, use commit instead.